manager/src/main/java/com/redhat/service/smartevents/manager/BridgesService.java

```diff
@@ -25,7 +26,7 @@ public interface BridgesService {
 
     List<Bridge> findByShardIdWithReadyDependencies(String shardId);
 
-    Bridge updateBridge(BridgeDTO bridgeDTO);
+    Bridge updateBridgeStatus(BridgeStatusWrapperDTO statusWrapperDTO);
```

Comment:
Just making the method name consistent with that used for `Processor`'s (and closer to what it actually does).